stream-dbs-clients/pom.xml

```diff
@@ -134,8 +139,15 @@
                                     <outputDirectory>${project.build.directory}/yaml</outputDirectory>
                                     <overWrite>true</overWrite>
                                 </artifactItem>
+                                <artifactItem>
+                                    <groupId>com.backbase.identity</groupId>
+                                    <artifactId>identity-integration-spec</artifactId>
+                                    <type>jar</type>
+                                    <outputDirectory>${project.build.directory}/raml/identity-integration</outputDirectory>
+                                    <overWrite>true</overWrite>
+                                </artifactItem>
                             </artifactItems>
-                            <includes>**/*.yaml, **/*.json</includes>
+                            <includes>**/*.yaml, **/*.json, **/*.raml, **/*.md</includes>
```

Comment:
do we really need raml here?

Reply:
we need it to export it to open api. don't know if theres a better method

Reply:
this is the wrong dependency, you should be using identity-integration-outbound-api-v1.0.1.yaml from 
```xml
          <groupId>com.backbase.identity</groupId>
          <artifactId>identity-integration-service</artifactId>
          <type>zip</type>
          <classifier>api</classifier>
```